notes/stories/film/film-scifi-1990s.st.txt

```diff
@@ -4135,7 +4135,7 @@ the desire for vengeance [Nea sought to avenge the slaughter of her family by th
 
 :: Minor Themes
 mother and daughter [Nea mother pleaded with her to flee from the cyborgs with her baby brother.]
-superstition [The primitive squatter camp people presumed that the marauding cyborgs were demons.]
+modern superstition [The primitive squatter camp people presumed that the marauding cyborgs were demons.]
```

Comment:
Are demons too a modern superstition? I'm beginning to think we have rather different ideas about modernity...!

Reply:
That was me with a mind to change them all and sort them out later. It's chanted to *primitive point of view* though for now.